src/frontend/e2e/page-objects/LoginPage.ts

```diff
@@ -0,0 +1,29 @@
+import {expect, Locator, Page} from '@playwright/test'
+
+
+export class LoginPage{
+
+    readonly page: Page
+    readonly signButton: Locator
+    readonly usernameInput: Locator
+    readonly passwordInput: Locator
+    readonly continueButton: Locator
+
+    constructor(page: Page){
+        this.page = page
+        this.signButton = page.locator('a[data-test-id="navbar-sign-in-link"]');
+        this.usernameInput = page.locator('#username');
+        this.passwordInput = page.locator('#password');
+        this.continueButton = page.locator("main.login button[type='submit']");
+    }
+
+    async login(username: string, password: string){
+        await this.page.goto('https://staging.czgenepi.org/');
```

Comment:
We should not hard code baseUrls; this will not run against dev/ci or other environments. For now leave it; we will change once my PR that implements global setup is merged

Reply:
To be fixed in another PR